optimum/onnxruntime/modeling_ort.py

```diff
@@ -131,7 +130,7 @@ def load_model(path: Union[str, Path], provider=None):
                 ONNX Runtime provider to use for loading the model. Defaults to `CPUExecutionProvider`.
         """
         if provider is None:
-            provider = "CPUExecutionProvider"
+            provider = "CUDAExecutionProvider" if _is_gpu_available() else "CPUExecutionProvider"
```

Comment:
I've been using TensorRT on another branch successfully. I'm curious how we imagine being able to pass that as an option to the InferenceSession

https://github.com/huggingface/optimum/compare/main...sam-h-bean:optimum:mrbean/tensorrt?expand=1